gitlab/v4/objects/snippets.py

```diff
@@ -12,6 +13,11 @@
 from .discussions import ProjectSnippetDiscussionManager  # noqa: F401
 from .notes import ProjectSnippetNoteManager  # noqa: F401
 
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
```

Comment:
I'd likely do this in a try/except block as it used to be done for most py2/3 imports, but see my other comment about potentially getting rid of it - sorry and thanks for your patience :)

Reply:
I tried this using a try/except block at first but mypy complained:
```
error: Incompatible import of "Literal" (imported name has type "typing_extensions._SpecialForm", local name has type "typing._SpecialForm")
```
Then I found this in the mypy docs: https://mypy.readthedocs.io/en/stable/runtime_troubles.html#using-new-additions-to-the-typing-module which seems to work ok. 

If you know a way to get it to play nice with try/except I'd be happy to do that too of course.